### DISCUSSION TITLE
Change Search Box Color?

### DISCUSSION THREAD

**Discussion by StodgyWaif** [Open-Shell/Open-Shell-Menu#1695] [Q&A]
Hello I am using the immersive skin.
Is it possible to change the search box color?
I am using Win11 with explorerpatcher
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/25680410/4980efcc-6522-41bb-8c4f-8f81f95dc38e)

**Comment by ge0rdi** [comment]
> Is it possible to change the search box color?

Search box background is determined by skin `Menu theme` setting. So it can only be either dark or light.

If you want custom color then you'll have to change the skin file.
This is quite advanced topic, you can have a look at skinning tutorial in `Open-Shell` help.

**Reply by StodgyWaif** [reply]
Thank you. I did try modifying the skin but could not find correct entry to change.
Can anyone point me in the right direction? I'd prefer the box to be white or colored the same as the rest.

**Reply by EJSnow** [reply]
You can try [Fluent-Metro](https://github.com/bonzibudd/Fluent-Metro). It has a lot of options, but you can set the search box color to white or dark or auto light/dark, or your accent color. Just a couple changes to its settings and you can get it looking almost the same as the Immersive skin. Oh, and it has a cool gradient effect on selectors you can enable.
It really does have a lot of options, though, and if you prefer a Windows 10 style menu, it requires a number of tweaks to get it looking right, and some things (read: shutdown button) aren't restyled with the rest of the items. But if you're into that kind of thing, go for it.

**Reply by EJSnow** [reply]
I can also look into Fluent-Metro's skin file to figure out how the recoloring of the search box background works.
